rustls/tests/runners/macros.rs

```diff
@@ -1,37 +1,45 @@
-/// Instantiate the given test functions once for each built-in provider.
-///
-/// The selected provider module is bound as `provider`; you can rely on this
-/// having the union of the public items common to the `rustls::crypto::ring`
-/// and `rustls::crypto::aws_lc_rs` modules.
+//! Instantiate the given test functions once for each built-in provider.
```

Comment:
I think this headline might need tweaking. WDYT about something like:  "Import and scaffold a built-in crypto provider" (?)

Reply:
Reworded.